go.mod

```diff
@@ -196,7 +196,7 @@ require (
 	github.com/cilium/ebpf v0.16.0
 	github.com/clbanning/mxj v1.8.4
 	// pin the commit adding `no_dynamic_plugins` build tag, can be updated whenever a new release is out
```

Comment:
The needed commit is now present in `1.7.25`: https://github.com/containerd/containerd/commit/2b12ef2f421f141805f8afcd72d1315698b2582c

```suggestion
```